### DISCUSSION TITLE
Real estate feedback of the front page emphasized based on what is relevant

### DISCUSSION THREAD

**Discussion by ilayn** [community/community#37574] [Other Feature Feedback, Questions, & Ideas] [answered]
This is a depiction of what is important for me and the irrelevant stuff covered. 

I don't remember why or how I got into contact with those covered entries but I'm not interested in what they do. I come to github to do open source stuff I am involved with. You are practically forcing me to unfollow everyone I don't know. 

I am also not sure why following is still a thing in GitHub but I am an old enough member to remember that there was a time where people on the same repo was mentioned in the front page which was way better than the last 3 versions of the homepage.

![image](https://user-images.githubusercontent.com/1303842/198416055-252d139e-cfa5-4639-8d25-48d6de1e2a7b.png)

**Comment by ossanna16** [comment]
Hi there @ilayn and welcome to our community! Thank you for the feedback 🙂

**Comment by ilayn** [comment, marked as answer]
Anyways, clearly, Github is following Microsoft techniques such as remaining silent and not taking action so closing this one.

**Reply by siddhpant** [reply]
True. I also don't know why I see ossanna16 commenting the same thing in many places...

**Reply by falsandtru** [reply]
> Hi there @ilayn and welcome to our community! Thank you for the feedback :slightly_smiling_face:

This reply means he thinks "this user doesn't want a response and we don't have to respond". It looks like he just wanted to create an alibi in response to us and change the discussion at the top of the page.
